cmd/crane/README.md

```diff
@@ -13,7 +13,7 @@ A collection of useful things you can do with `crane` is [here](recipes.md).
 Download [latest release](https://github.com/google/go-containerregistry/releases/latest):
 
 ```
-$ VERSION=TODO   # Latest, or other
+$ VERSION=TODO   # latest, or other
```

Comment:
This is actually correct. It's intended to be a sentence, essentially "Use the latest release, or another one". No relation to the `:latest` tagging convention or Go's `@latest` convention.

If `VERSION=latest` (or `VERSION=Latest` for that matter), the `curl` command below will fail since neither of those are release names.